tmpfiles.d/systemd.conf.in

```diff
@@ -60,6 +60,9 @@ a+ /var/log/journal/%m/system.journal - - - - group:wheel:r--
 
 d /var/lib/systemd 0755 root root -
 d /var/lib/systemd/coredump 0755 root root 3d
+# Files and directories in /var/lib/systemd/ephemeral are locked by pid 1 to prevent tmpfiles from removing
+# them, so we tell tmpfiles to clean up anything in /var/lib/systemd/ephemeral unconditionally.
+d /var/lib/systemd/ephemeral 0755 root root 0
```

Comment:
a wild.

can you add a tmpfiles.d(5) man page about this behaviour. It's a really useful concept i think. i.e. mention that age 0 makes sense if combined with locking